docs/Config.md

```diff
@@ -65,6 +65,7 @@ git:
   parseEmoji: false
 os:
   editCommand: '' # see 'Configuring File Editing' section
+  editCommandTemplate: '{{editor}} {{filename}}'
```

Comment:
Can you add documentation on what the differences between these 2 are?

Reply:
I've added some documentation, but my English isn't very good.